bin/resty

```diff
@@ -130,7 +130,15 @@ my $conns = $conns_num || 64;
 my @nameservers;
 
 if (!$resolve_ipv6) {
-    unshift @nameservers, "ipv6=off";
+    # no error check to let the later exec() fail.
+    # worst case: ipv6=off is not added to the nginx
+    # conf, but the worker won't spawn anyways...
+    my $stderr = `$nginx_path -V 2>&1 1>/dev/null`;
+    #warn $stderr;
+
+    if (index($stderr, "--with-ipv6") != -1) {
```

Comment:
Given some more thoughts about this. How about:

* `< 1.11.5`
  * we perform the `--with-ipv6` check
    * support enabled
      * no `--resolve-ipv6` flag: we add the `ipv6=off` option
      * `--resolve-ipv6` flag: do nothing, it is nginx's default behavior
    * support disabled
      * no `--resolve-ipv6` flag: do nothing
      * `--resolve-ipv6` flag: error out: "No support for IPv6 resolution but --resolve-ipv6 flag"
* `>= 1.11.5`
  * do nothing: regardless of nginx being built with or without support for IPv6, the default behavior is what we seem to be looking for.